arangodb-net-standard/CollectionApi/CollectionApiClient.cs

```diff
@@ -139,7 +140,7 @@ public async Task<GetCollectionResponse> GetCollectionAsync(string collectionNam
         /// <returns></returns>
         public async Task<GetCollectionPropertiesResponse> GetCollectionPropertiesAsync(string collectionName)
         {
-            using (var response = await _transport.GetAsync(_collectionApiPath + "/" + collectionName + "/properties"))
+            using (var response = await _transport.GetAsync(_collectionApiPath + "/" + WebUtility.UrlEncode(collectionName) + "/properties"))
```

Comment:
Some lines end up longer than 100 characters. Should we build the URI in a separate variable?